src/bindings/python/src/openvino/frontend/pytorch/__init__.py

```diff
@@ -13,6 +13,7 @@
     from openvino.frontend.pytorch.py_pytorch_frontend import _Type as DecoderType
     from openvino.frontend.pytorch.py_pytorch_frontend import ConversionExtensionPytorch as ConversionExtension
     from openvino.frontend.pytorch.py_pytorch_frontend import OpExtensionPytorch as OpExtension
+    from openvino.frontend.pytorch.module_extension import ModuleExtension
 except ImportError as err:
     raise ImportError("OpenVINO PyTorch frontend is not available, please make sure the frontend is built."
                       "{}".format(err))
```

Comment:
please update documentation `How to convert PyTorch models` and demonstrate how to use it

Reply:
I don't think this will be the right place for this documentation, this functionality is not meant for regular users

Reply:
It gives much clarity about what feature you provide to users and easier to review from arch point.

Reply:
Let us update at least ovc documentation in ovc code.